src/tests/dictionaryConcepts/container/CreateConcept.test.jsx

```diff
@@ -115,6 +120,33 @@ describe('Test suite for dictionary concepts components', () => {
     jest.runAllTimers();
   });
 
+
+  it('should handle add-more-answers', () => {
+    const wrapper = mount(<Router>
+      <CreateConcept {...props} />
+    </Router>);
+    wrapper.find('.add-more-answers').simulate('click');
+    expect(props.addNewAnswer).toHaveBeenCalled();
+  });
+
+  it('should handle remove-answers', () => {
+    const wrapper = mount(<Router>
+      <CreateConcept {...props} />
+    </Router>);
+    wrapper.find('.concept-form-table-link.answer').simulate('click');
+    expect(props.removeAnswer).toHaveBeenCalled();
+  });
+
+  it('should handle add data from answer', () => {
+    const wrapper = mount(<Router>
+      <CreateConcept {...props} />
+    </Router>);
+    const conceptAnswer = { target: { name: 'answer', value: 'test concept' } };
+    const conceptAnswer2 = { target: { name: 'answer', value: 'test concept 3445' } };
+    wrapper.find('#concept-answer').simulate('change', conceptAnswer);
+    wrapper.find('#concept-answer').simulate('change', conceptAnswer2);
```

Comment:
What is this test doing?

Reply:
I have refactored this test please check it out.